x-pack/plugins/security_solution/public/common/components/event_details/table/summary_value_cell.tsx

```diff
@@ -25,32 +27,36 @@ export const SummaryValueCell: React.FC<AlertSummaryRow['description']> = ({
   timelineId,
   values,
   isReadOnly,
-}) => (
-  <>
-    <FieldValueCell
-      contextId={timelineId}
-      data={data}
-      eventId={eventId}
-      fieldFromBrowserField={fieldFromBrowserField}
-      linkValue={linkValue}
-      isDraggable={isDraggable}
-      style={{ flexGrow: 0 }}
-      values={values}
-    />
-    {timelineId !== TimelineId.active && !isReadOnly && !FIELDS_WITHOUT_ACTIONS[data.field] && (
-      <ActionCell
+}) => {
+  const hoverActionsEnabled = !FIELDS_WITHOUT_ACTIONS[data.field];
```

Comment:
👍  extracting the inline condition and moving it here improves readability
✅  already covered by a unit test